Makefile

```diff
@@ -139,6 +140,13 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
+add-copyright-header: ## Add copyright header to all files in the project
+ifeq (, $(shell which copywrite))
+	@echo "Installing copywrite"
+	@go install github.com/hashicorp/copywrite@latest
+endif
+	@copywrite headers --spdx "MPL-2.0" 
```

Comment:
This makes using `copywrite` the tool we use for adding copyright headers easier. It's a no-op for already copyrighted files.

One of the issues I was running into when generating the CRD YAML was that they lacked the necessary header. This script can now be executed after the generation to prevent that problem.